syntax = "proto2";
package pb;

message NoiseExtensions {
    repeated string stream_muxers = 2;
}

message NoiseHandshakePayload {
	optional bytes identity_key = 1;
	optional bytes identity_sig = 2;
	optional bytes data = 3;
}
